### ISSUE TITLE
sqlite errors

### ISSUE THREAD

**Issue by ofirgalcon** [tuxudo/tcc#4]
I am getting this on my test machine with 12.3.1

```
Traceback (most recent call last):
  File "/usr/local/munkireport/scripts/tcc", line 184, in <module>
    main()
  File "/usr/local/munkireport/scripts/tcc", line 175, in main
    result = process_dbs(dbs)
  File "/usr/local/munkireport/scripts/tcc", line 43, in process_dbs
    for entry in tcc.read_db():
  File "/usr/local/munkireport/scripts/tcc", line 150, in read_db
    query = self.sqlite.query('SELECT service, client, client_type, auth_value, auth_reason, indirect_object_identifier, last_modified FROM access', fetch=True)
  File "/usr/local/munkireport/scripts/tcc", line 124, in query
    self.c.execute(query_string)
sqlite3.OperationalError: no such column: auth_value
```

**Comment by tuxudo** [comment]
What version of the module is that and what version of macOS? That can also happen if MunkiReport cannot read the TCC database due to TCC permissions.

**Comment by ofirgalcon** [comment]
12.3.1 and the very latest version you released - 2.4.

I believe i gave the tcc script the correct permissions. I signed it and then dragged it into the Security & Privacy window

<img width="780" alt="image" src="https://user-images.githubusercontent.com/16665880/166109056-2aba2f0c-9f2f-4d40-9a7e-e27fb8cc36ad.png">

Maybe I'm doing something wrong but I was never able to get this module to work

**Comment by tuxudo** [comment]
Can you DM me the `/Library/Application Support/com.apple.TCC/TCC.db` and `/Users/$current_user/Library/Application Support/com.apple.TCC/TCC.db` from that Mac on Slack? I'll look into why it's failing for that. Everything looks to be working for you with that setup. 

If you run Jamf, you can use the `tcc_jamf` module without having to deal with TCC itself: https://github.com/tuxudo/tcc_jamf

**Comment by tuxudo** [comment]
Resolved with version v2.5 of module.
